velox/functions/prestosql/tests/StringFunctionsTest.cpp

```diff
@@ -1862,3 +1862,34 @@ TEST_F(StringFunctionsTest, varbinaryLength) {
   auto result = evaluate("length(c0)", makeRowVector({vector}));
   test::assertEqualVectors(expected, result);
 }
+
+TEST_F(StringFunctionsTest, normalize) {
+  const auto normalizeWithoutForm =
+      [&](std::optional<std::string> string) {
+        return evaluateOnce<std::string>("normalize(c0)", string);
+      };
+
+  const auto normalizeWithForm =
+      [&](std::optional<std::string> string, std::optional<std::string> form) {
+        return evaluateOnce<std::string>("normalize(c0, c1)", string, form);
+      };
+
+  // EXPECT_EQ(normalizeWithoutForm(std::nullopt), std::nullopt);
```

Comment:
I guess you didn't want to have these commented out?

Reply:
Yes, I am working on the changes still , don't review yet. Thanks.

Reply:
Yes, an empty string should return an empty string.